dataset_config/template/dataset.json

```diff
@@ -1,6 +1,11 @@
 {
-  // Dataset name. For Project Baseline (https://www.projectbaseline.com/),
-  // this could be "Project Baseline". This name will be displayed in the Data
-  // Explorer UI. Also, the Elasticsearch index will be given this name.
-  "name": ""
+  // Dataset name. This name will be displayed in the Data Explorer UI. Also,
```

Comment:
nit: I'm really not a fan of how bad this looks on github (also in my IDE) from a style/presentation point of view. Definitely think the comments are useful, and minifying before parsing is the way to go, but I think them being highlighted red like this really detracts form their usefulness.

It seems to only show up like this if the comments are within the initial first braces ([example](https://github.com/DataBiosphere/data-explorer/blob/master/dataset_config/1000_genomes/bigquery.json#L1)). If we made the comments a little more verbose and put them at the top I think it would look significantly better. Something like:
```
// Configuration metadata for the data set this data explorer references.
// Config Fields:
//   name: 
//     The dataset name which will be displayed in the DataExplorer UI, also the name of the elasticsearch index.
//   authorization_domain:
//     The FireCloud authorization domain. This must be set for access-controlled datasets. It is used to restrict 
//     which Saturn workspaces data can be sent to. See 
//     https://gatkforums.broadinstitute.org/firecloud/discussion/9524/authorization-domains

{
    "name": "",
    "authorization_domain": ""
}
```

WDYT?